pkg/rancher-desktop/backend/lima.ts

```diff
@@ -752,13 +752,24 @@ export default class LimaBackend extends events.EventEmitter implements VMBacken
     const VMNETDir = path.join(VMNET_DIR, 'bin');
     const pathList = (process.env.PATH || '').split(path.delimiter);
     const newPath = [binDir, VMNETDir].concat(...pathList).filter(x => x);
+    const env = structuredClone(process.env);
+
+    env['LIMA_HOME'] = paths.lima;
+    env['PATH'] = newPath.join(path.delimiter);
+
+    // Override LD_LIBRARY_PATH to pick up the QEMU libraries.
+    // - on macOS, this is not used. The macOS dynamic linker uses DYLD_ prefixed variables.
+    // - on packaged (rpm/deb) builds, we do not ship this directory, so it does nothing.
+    // - for AppImage this has no effect because the libs are moved to a dir that is already on LD_LIBRARY_PATH
+    // - this only has an effect on builds extracted from a Linux zip file (which includes a bundled
+    //   QEMU) to make sure QEMU dependencies are loaded from the bundled lib directory.
+    if (env['LD_LIBRARY_PATH']) {
+      env['LD_LIBRARY_PATH'] = `${ libDir }:${ env['LD_LIBRARY_PATH'] }`;
```

Comment:
That's technically [`path.delimiter`](https://nodejs.org/docs/latest-v18.x/api/path.html#pathdelimiter) but I suspect we don't care. (We actually used it for `$PATH` above.)